Lib9c/Action/BuyMultiple.cs

```diff
@@ -0,0 +1,396 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Libplanet.Assets;
+using Nekoyume.Model.EnumType;
+using Nekoyume.Model.Item;
+using Nekoyume.Model.Mail;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Serilog;
+
+namespace Nekoyume.Action
+{
+    [Serializable]
+    [ActionType("buy_multiple")]
+    public class BuyMultiple : GameAction
+    {
+        public Address buyerAvatarAddress;
+        public IEnumerable<PurchaseInfo> purchaseInfos;
+        public BuyerResult buyerResult;
+        public SellerResult sellerResult;
+
+        public const int FAILED_LOADING_STATE = 1;
+        public const int ITEM_DOES_NOT_EXIST = 2;
+        public const int SHOPITEM_EXPIRED = 3;
+        public const int INSUFFICIENT_BALANCE = 4;
```

Comment:
```suggestion
        public const int PURCHASE_FAILED_LOADING_STATE = 1;
        public const int PURCHASE_FAILED_ITEM_DOES_NOT_EXIST = 2;
        public const int PURCHASE_FAILED_SHOPITEM_EXPIRED = 3;
        public const int PURCHASE_FAILED_INSUFFICIENT_BALANCE = 4;
```
These fields can be grouped by several ways as above.

Reply:
Seems like a good idea, and I grouped with `ERROR_CODE`.